### ISSUE TITLE
The src was renamed to scrlib

### ISSUE THREAD

**Issue by besser** [sourcegraph/sourcegraph-atom#25]
http://sourcegraph.com/github.com/sourcegraph/srclib/.commits/405e375e4b39af18396ac363b780f3cd1137f90a

**Comment by MaikuMori** [comment]
I believe editor support was removed at the same time so we'll have to wait for it to be added back. A simple rename will not work.

**Comment by beyang** [comment]
That's correct, the existing editor plugins are currently not supported, but we are working hard to update the srclib schema and API to support more advanced features (like autocompletion) in order to have much better editor support in the future.
